app/views/tag/show/_tab_content.html.erb

```diff
@@ -7,8 +7,8 @@
   <% if @node_type == "note" %>
     <div class="tab-pane active" id="notes">
       <% if @notes.nil? || @notes.length == 0 %>
-        <p><%= raw translation('tag.show.no_results_found', tag: params[:id]) %>:</p>
-        <p><a href="/search/<%= params[:id] %>">Search for "<%= params[:id] %>"</a></p>
+        <p><%= raw translation('tag.show.no_results_found', other_tab: 'Wiki') %></p>
```

Comment:
@jywarren Instead of using the en.yml to display the message, I think maybe we should just display the message here since each tab has it's own no_results found message.